### PR TITLE
Fix bug in image check ansible task

### PR DESCRIPTION
**Pull request by patricksimonian**
this should address red herring errors on new deployments.

### CODE REVIEW

File: ansible/tasks/deploy_react.yaml [hunk 1]
```diff
@@ -6,8 +6,11 @@
     vars:
       IMAGE_TAG: '{{ version }}'
       deploy_params: "{{ lookup('template', './templates/deploy_react.param.j2') }}"
+  - debug: msg="Image Tag has not been created yet or the build is still in progress. A rollout has been triggered and may fail a few times before be successful"
+    when: not image_exists
```
Comment (thegentlemanphysicist) [inline ansible/tasks/deploy_react.yaml:10]: is this indenting correct?  I'm not that familiar with yaml, but should the conditional be at a lower indentation level than the logic?

Comment (patricksimonian) [reply ansible/tasks/deploy_react.yaml:10]: Yep its correct. That python indentation. Consider YAML more of JSON derivative. This is describing key/value pairs inside of an object
```json
[
 {
   "debug": "...",
   "when": "..."
 }
]
```

is equivalent

File: ansible/group_vars/all/vars.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 tools_namespace: xdwiwd-tools
 dev_namespace: xdwiwd-dev
 test_namespace: xdwiwd-test
-prod_namespace: xdwiwd-tools
+prod_namespace: xdwiwd-prod
```
Comment (thegentlemanphysicist) [inline ansible/group_vars/all/vars.yaml:5]: How was this able to successfully deploy prod the first time this was built? Was it hosting prod in the tools namespace?

Comment (patricksimonian) [reply ansible/group_vars/all/vars.yaml:5]: I think we had a false positive :) :(

**Pull request merged**